tests/D2L.CodeStyle.Analyzers.Test/Specs/DependencyRegistrationsAnalyzer.cs

```diff
@@ -25,6 +25,9 @@ public enum ObjectScope {
 	public abstract class ExtensionPointDescriptor {
 		public abstract string Name { get; }
 	}
+
+	[Singleton]
+	public interface IMarkedSingleton { }

```

Comment:
Can you delete this instead?  It looks like Saquib replaced all the references to `IMarkedSingleton` with `ISingleton` in this commit:
https://github.com/Brightspace/D2L.CodeStyle/commit/46d0599443752a888c0e1b9ac379bc3111c19634#diff-28d8752f46afac5feb152b15935f80b9
That change made sense.  If there were any references left behind, updating to ISingleton makes more sense than bringing the Marked/Unmarked stuff back